examples/site.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: all
+
+  pre_tasks:
+    - name: Update APT cache
+      apt: update_cache=yes
+
+  roles:
+    - { role: "azavea.git" }
```

Comment:
`dependencies` in `meta` should pull these in automatically with the `azavea.opentripplanner` role.